src/LibChorus/MercurialLocation.cs

```diff
@@ -42,12 +47,14 @@ public static string PathToMercurialFolder
 		}
 
 		/// <summary>
-		/// Will use the PathToMercurialFolder, otherwise will just return "hg"
+		/// Will use the environment variable override if set, or will use PathToMercurialFolder if possible, otherwise will just return "hg" and rely on PATH
 		/// </summary>
 		public static string PathToHgExecutable
 		{
 			get
 			{
+				string path = Environment.GetEnvironmentVariable(EnvPathToHgExecutable);
+				if (path != null) return path;
```

Comment:
this should probably use a null or empty check

Reply:
Fixed in commit 179726ab.